drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fmac_api.h

```diff
@@ -120,8 +120,7 @@ enum wifi_nrf_status nrf_wifi_fmac_rf_test_rx_cap(struct wifi_nrf_fmac_dev_ctx *
  * nrf_wifi_fmac_rf_test_tx_tone() - Start/Stop RF TX tone test in radio test mode.
  * @fmac_dev_ctx: Pointer to the UMAC IF context for a RPU WLAN device.
  * @enable: Enable/Disable TX tone test.
- * @norm_frequency: Desired tone frequency.
- * @tone_amplitude: Desired tone amplitude in the range 0 to 1023.
+ * @tone_freq: Desired tone frequency.
```

Comment:
Add more description-
Desired tone frequency in MHz in steps of 1MHz from -10MHz to +10MHz.